### PR TITLE
Fix Python Dockerfile

### PR DESCRIPTION
**Pull request by GyuminJack**
dockerfile is updated 2 month ago.

there is missed change directory command. (it is originally existed)

### CODE REVIEW

File: docker/dockerfile-python [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN apt-get update && \
     # lightgbm
     conda install -q -y numpy scipy scikit-learn pandas && \
     git clone --recursive --branch stable --depth 1 https://github.com/Microsoft/LightGBM && \
-    sh ./build-python.sh install && \
+    cd LigthGBM && sh ./build-python.sh install && \
```
Comment (jameslamb) [inline docker/dockerfile-python:29]:
```suggestion
    cd ./LightGBM && \
    sh ./build-python.sh install && \
```

Please put this change on a separate line, for consistency with the rest of this Dockerfile (which is done to make it easier to see the sequence of commands).

And please note that it's `LightGBM`, not `LihgtGBM`.

**Review by jameslamb — CHANGES_REQUESTED** [review]
Thanks for taking the time to contribute!

Please see one suggestion I left. In the future, we'd appreciate if you tested your changes prior to submitting them.

**Comment by GyuminJack** [comment]
I misspelled it when I typed it in the source after testing it on something else.
I apologize. (i tested twice this time..😅)

Next time I contribute, I'll be sure to test or build against the changed source and open a PR.

Thanks for the thoughtful comment.

**Comment by jameslamb** [comment]
No problem, thanks for the fix!

By the way, please check you gitconfig settings. It looks like you're pushing commits here that aren't signed with an email tied to your GitHub account. That means GitHub might not include you as a "contributor" in this project's statistics, and people won't be able to click through to your account from the commit log or git blame view.

See https://github.com/microsoft/LightGBM/pull/5532#issuecomment-1274032708 for some details on what I mean.

**Comment by GyuminJack** [comment]
i did below code. ( ref, https://github.com/uptake/pkgnet/pull/284)
```
git config --global user.email "my email"
git reset --soft HEAD~2
git commit -m "fix commit message"
git push origin master --force
```

Is this what you were expecting?

**Comment by jameslamb** [comment]
Looks like that fixed it! The commits are now tied to your GitHub.

Screen Shot 2023-07-17 at 8 15 43 PM

> Is this what you were expecting?

It doesn't matter to me personally, we could have merged this either way. Just want to be sure you get credit for your contribution 😊

**Review by jameslamb — APPROVED** [review]

**Comment by jameslamb** [comment]
thanks!

**Pull request merged**